### PR TITLE
feat: added stats at subsection level in topics tab

### PR DESCRIPTION
**Pull request by muhammadadeeltajamul**
### Description

Added stats at sub-section level in topics tab.

Ticket: [INF-763](https://2u-internal.atlassian.net/browse/INF-763)

#### How Has This Been Tested?

Open Topics tab in discussions mfe. Stats will be visible for courseware topics

#### Visual Change Screenshots
Before
inf-763_before

After
inf-763_after

#### Merge Checklist

* [x] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Sandbox, if applicable.
* [x] Is there adequate test coverage for your changes?

### CODE REVIEW

**Comment by mehaknasir** [comment]
Can we add test cases for new components please?

**Comment by muhammadadeeltajamul** [comment]
Test cases for v3 topics are in progress in another ticket

File: src/data/constants.js [hunk 1]
```diff
@@ -212,3 +214,15 @@ export const ALL_ROUTES = []
...
+  id: PropTypes.string,
+  usage_key: PropTypes.string,
+  name: PropTypes.string,
+  thread_counts: PropTypes.shape({
+    discussions: PropTypes.number,
+    questions: PropTypes.number,
+  }),
+  enabled_in_context: PropTypes.bool,
+  flags: PropTypes.number,
+});
```
Comment (awais-ansari) [inline src/data/constants.js:228]: `topicShape` is not a constant. Move this to the relevant component and then export from there.

**Review by awais-ansari — CHANGES_REQUESTED** [review]
Add a proper description, and before, and after screenshots for better visibility.

**Review by mehaknasir — APPROVED** [review]

**Review by awais-ansari — APPROVED** [review]
LGTM

**Pull request merged**